tests/filecheck/dialects/llvm/arithmetic.mlir

```diff
@@ -1,47 +1,66 @@
 // RUN: XDSL_ROUNDTRIP
 
-builtin.module {
-    %arg0, %arg1 = "test.op"() : () -> (i32, i32)
+%arg0, %arg1 = "test.op"() : () -> (i32, i32)
 
-    %add = llvm.add %arg0, %arg1 : i32
-    // CHECK: %add = llvm.add %arg0, %arg1 : i32
+%add_nsw = llvm.add %arg0, %arg1 overflow<nsw> : i32
+// CHECK: %add_nsw = llvm.add %arg0, %arg1 overflow<nsw> : i32
 
-    %add2 = llvm.add %arg0, %arg1 {"nsw"} : i32
-    // CHECK: %add2 = llvm.add %arg0, %arg1 {"nsw"} : i32
+%add_nuw = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<nuw>} : i32
+// CHECK: %add_nuw = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<nuw>} : i32
 
-    %sub = llvm.sub %arg0, %arg1 : i32
-    // CHECK: %sub = llvm.sub %arg0, %arg1 : i32
+%add_none = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<none>} : i32
+// CHECK: %add_none = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<none>} : i32
 
-    %mul = llvm.mul %arg0, %arg1 : i32
-    // CHECK: %mul = llvm.mul %arg0, %arg1 : i32
+%add_both = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<nsw, nuw>} : i32
+// CHECK: %add_both = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<nsw,nuw>} : i32
 
-    %udiv = llvm.udiv %arg0, %arg1 : i32
-    // CHECK: %udiv = llvm.udiv %arg0, %arg1 : i32
+%add_both_reverse = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<nuw, nsw>} : i32
+// CHECK: %add_both_reverse = llvm.add %arg0, %arg1 {"overflowFlags" = #llvm.overflow<nsw,nuw>} : i32
```

Comment:
I think we should be safe to remove these, the tests below with the custom syntax cover it enough, IMO

Reply:
Okay
I still kept one test with the complete syntax